src/less/frontend.less

```diff
@@ -301,20 +299,52 @@ body.full-width {
 }
 
 #sidenav-toggle {
-	float: left;
-	@media (max-width: 765px) {
-		padding-left: 15px;
-	}
+	color: white;
+	background-color: #3E3F3A;
```

Comment:
Hardcoding colors means they will not be affected when compiling the theme with different values in variables.less as gstreamer does for example. In that case I think you want to use the same color as the navbar's background, it is named @navbar-default-bg, which should translate to:

background-color: @navbar-default-bg

you'll want to do the same for the color of the text, instead of white I suggest you give a quick look at either hotdoc/less/variables.less , or in this file at the way we define the color for text in the sitenav :)